trigger.md

```diff
@@ -0,0 +1,108 @@
+# Finite Triggered Acquisition
+Acquire (`acquire-imaging` [on PyPI](https://pypi.org/project/acquire-imaging/)) is a Python package providing a multi-camera video streaming library focused on performant microscopy, with support for up to two simultaneous, independent, video streams.
+
+This tutorial shows an example of setting up triggered acquisition of a finite number of frames with one of Acquire's supported devices and saving the data to a Zarr file.
+
+## Initialize Acquisition
+
+To start, we'll import `Acquire` and create an acquisition `Runtime` object, which initializes the driver adaptors needed for the supported cameras.
+
+```python
+import acquire
+runtime = acquire.Runtime()
+```
+
+## Configure Camera
+
+All camera settings can be captured by an instance of the `Properties` class, which will be associated with a given camera acquisition. The settings can be stored in a dictionary (e.g: `Properties.dict()`). These settings can be saved to a JSON file to be subsequently loaded, (e.g. ` Properties(**json.load('acquire.json'))` ), using the [json library](https://docs.python.org/3/library/json.html#).
+
+```python
+props = runtime.get_configuration()
+
+import json
+with open("/path/to/acquire.json", "w") as f:
+    json.dump(props.dict(), f)
+```
+
+The current configuration settings can be checked and assigned to an instance of the `Properties` class with:
+
+```python
+p = runtime.get_configuration() 
```

Comment:
For consistency with the code block beginning on line 19, I would consider renaming all instances of `p` to `props`, especially since you refer to `props` on line 33.

Reply:
I agree. Updated in the latest commit.